### PR TITLE
New package: SDL_gfx-2.0.25

### PR DESCRIPTION
**Pull request by pullmoll**
Again, hopefully proper now.

### CODE REVIEW

**Comment by pullmoll** [comment]
It looks like the comparison really has to be ${sourcepkg}>=${version}_${revision} to actually work. I stumbled upon this by trying to actually use SDL_gfx-devel in the coming srcpkgs/widelands/template.

**Comment by xtraeme** [comment]
The comparision will work with a version comparator or a fixed version.

File: srcpkgs/SDL_gfx/template [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Template file for 'SDL_gfx'
+pkgname=SDL_gfx
+version=2.0.25
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="SDL-devel"
+short_desc="Graphics drawing primitves for SDL"
```
Comment (xtraeme) [inline srcpkgs/SDL_gfx/template]: typo "primitives"

**Comment by pullmoll** [comment]
@xtraeme who or what adds the line `libSDL_gfx.so.2.0.25 SDL_gfx-2.0.25_2` to common/shlibs?
Should I commit it with the package that needs SDL_gfx?

**Comment by dominikh** [comment]
The person contributing the package containing that file adds the line to the file. It's done in the commit that adds the package. I.e. you and unfortunately the commit that already got merged :) So yeah, fix that in a follow-up commit.

**Pull request merged**